checker/checker_breaking_test.go

```diff
@@ -288,13 +288,7 @@ func TestBreaking_OperationIdRemoved(t *testing.T) {
 
 	d, osm, err := diff.GetWithOperationsSourcesMap(getConfig(), &s1, &s2)
 	require.NoError(t, err)
-	errs := checker.CheckBackwardCompatibility(checker.GetChecks(utils.StringList{"api-operation-id-removed"}), d, osm)
-	for _, err := range errs {
-		require.Equal(t, checker.ERR, err.Level)
-	}
-	require.NotEmpty(t, errs)
-	require.Len(t, errs, 1)
-	require.Equal(t, "api-operation-id-removed", errs[0].Id)
+	verifyNonBreakingChangeIsChangelogEntry(t, d, osm, "api-operation-id-removed")
```

Comment:
According to the func comment this is a breaking change (ERR or WARN), but the test expects an INFO-level error.

Reply:
You are right, I confused this with the non-breaking test. I'll revert this and add another test in the not_breaking file